### PR TITLE
demo: Upload demo add margin

### PR DESCRIPTION
**Pull request by CooperHash**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [X] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

#48091 

### 💡 Background and solution

after remove margin to flex
Upload's avatar.tsx demo needs manual spacing adjustment.

before: 
image

after:
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    add margin to upload demo      |
| 🇨🇳 Chinese |  新增upload demo间距         |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

File: components/upload/demo/avatar.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const App: React.FC = () => {
   );
 
   return (
-    <>
+    <Space>
```
Comment (li-jia-nan) [inline components/upload/demo/avatar.tsx:52]: 建议用 Flex 组件：

```suggestion
    <Flex gap="middle" wrap="wrap">
```

**Comment by li-jia-nan** [comment]
需要更新一下快照

**Comment by li-jia-nan** [comment]
快照还是有问题，看一下 CI 哪里挂了

**Comment by CooperHash** [comment]
貌似是preview？

**Comment by CooperHash** [comment]
暂时没看出问题在哪

**Comment by li-jia-nan** [comment]
image

**Comment by li-jia-nan** [comment]
执行一下 `npm run test:update components/upload`

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**